package.json

```diff
@@ -16,7 +16,7 @@
     "lint": "eslint . && prettier -c .",
     "format": "eslint --fix . && prettier --write .",
     "cover": "test -d .nyc_output && nyc report --reporter lcov",
-    "test": "dot-only-hunter test && nyc mocha --recursive --timeout=30000 --exit"
```

Comment:
There were a lot of tests which didn't wait for the outcome, so their results were just ignored.
After fixing the tests and waiting for the test results, 30 seconds isn't enough to run all the tests. Each test has it's own timeout instead.